classes/python3-usr_checkout.yaml

```diff
@@ -0,0 +1,17 @@
+checkoutSetup: |
```

Comment:
Should we add a `checkoutTools: [python3]`? I guess there is no point in using this class without the pyhton3 tool...